app/modules/medication/index.js

```
@@ -0,0 +1,789 @@
+          await setMedicationActive(id, nextActive, {
+            dayIso: currentDayIso,
+            reason: nextActive ? 'card:reactivate' : 'card:archive'
+          });
+          updateFormStatus(
+            nextActive ? 'Medikament reaktiviert.' : 'Medikament archiviert.'
+          );
+          if (!nextActive && ui.id.value === id) {
+            clearForm();
+          }
+        });
+        return;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)